pages/manifesto.tsx

```diff
@@ -36,18 +62,45 @@ const Manifesto = () => {
   }, [])
 
   useEffect(() => {
-    ;(async () => {
+    const fetchAllSignatures = async () => {
```

Comment:
Although an IIFE (immediately invoked function expression) works here, for readability and consistency with the pattern above, switched to a named function.